gyro/robot.py

```diff
@@ -4,9 +4,9 @@
 from wpilib.drive import DifferentialDrive
 
 
-class MyRobot(wpilib.SampleRobot):
+class MyRobot(wpilib.IterativeRobot):
```

Comment:
Any particular reason to use IterativeRobot, given it's deprecated? (See also #23.)